parser.messages

```diff
@@ -140,9 +140,9 @@ program: OSR IDENTIFIER LBRACKET NIL RBRACKET LPAREN IDENTIFIER COMMA IDENTIFIER
 
 There was an error parsing the specification of the new environment of an
 osr instruction. The specification is a comma-separated list of terms of the form
-"const x = e" (where "e" is an expression), "mut x = e", "mut x = &y"
-or just "mut x". For example,
-"osr l [(a==1)] (Main,optimized,l2) [const x = e, mut y = &y, mut z]".
+"const x = e" (where "e" is an expression), "mut x = e", "mut x = &y"
+or just "mut x". For example,
+"osr l [(a==1)] (Main,optimized,l2) [const x = e, mut y = &y, mut z]".
```

Comment:
The changes look like something your editor would do automatically and they are wrong (they turn unbreakable spaces into breakable spaces). Could you revert the changes to `parser.messages`? (`git show master:parser.messages > parser.messages` would revert the file, and then you could commit that.)